plugins/src/android/optimove.gradle

```diff
@@ -19,7 +19,9 @@ dependencies{
     implementation("androidx.work:work-runtime-ktx:$work_version") {
         force = true
     }
-  
+    implementation platform('com.google.firebase:firebase-bom:28.2.0')
+    implementation 'com.google.firebase:firebase-messaging'
```

Comment:
firebase added 2 times? also, com.optimove.android:optimove-android:5.+ shall be 6.+, not sure if is updated somewhere else

Reply:
updated dependencies 👍